esp/envoy-config.yaml

```diff
@@ -38,7 +38,7 @@ static_resources:
                             prefix: "/"
                           route:
                             cluster: grpcserver
-                            timeout: 600s
+                            timeout: 60s
```

Comment:
Looks like we are bringing it down here and at a couple other places. Assuming this is expected?

Reply:
Do we want to drop those?  I worry if that might cause new timeout errors?

Reply:
As discussed offline, we should expect our APIs return in less than 1 min. Anything longer should indicate a performance issue and needs to be fixed.